pkg/volume/volume_linux.go

```diff
@@ -161,8 +161,12 @@ func requiresPermissionChange(rootDir string, fsGroup *int64, readonly bool) boo
 	unixPerms |= execMask
 	filePerm := fsInfo.Mode().Perm()
 
-	// We need to check if actual permissions of root directory is a superset of permissions required by unixPerms
-	// and setgid bits are set in permissions of the directory.
+	// We need to check if actual permissions of root directory is a superset of permissions required by unixPerms.
+	// This is done by checking if permission bits expected in unixPerms is set in actual permissions of the directory.
+	// We use bitwise AND operation to check set bits. For example:
+	//     unixPerms: 770, filePerms: 775 : 770&775 = 770
+	//     unixPerms: 770, filePerms: 770 : 770&770 = 770
```

Comment:
Also give an example where it's not a superset

Reply:
fixed